plugins/test/runner_main.cc

```diff
@@ -44,6 +46,10 @@ ABSL_FLAG(service_extensions_samples::pb::Env::LogLevel, loglevel,
           "Override log_level.");
 ABSL_FLAG(bool, test, true, "Option to disable config-requested tests.");
 ABSL_FLAG(bool, bench, true, "Option to disable config-requested benchmarks.");
+ABSL_FLAG(uint64_t, num_background_streams, 0,
+          "Number of background streams to run in benchmarks.");
+ABSL_FLAG(std::optional<uint64_t>, background_stream_advance_rate, std::nullopt,
```

Comment:
Why is this one optional, but the one above isn't? This must be >0 to make sense, so ignoring a default value of 0 seems OK to me. (no need to support disabling the advance feature IMO, since we have the StreamLifecycle benchmark to measure that behavior)

Reply:
Done